### PR TITLE
feat: nuxt

### PR DESCRIPTION
**Pull request by niceEli**
lol why not cuh we need bloat 👍🏻👍🏻👍🏻 also we want good build systerms and not stactick uhh we also use bun now cuh why not ig uhh idk seems peak

### CODE REVIEW

**Comment by niceEli** [comment]
@Dishpit @ronnie-rios

**Comment by niceEli** [comment]
uh isk how to make me asignee

**Comment by niceEli** [comment]
btw you should use squash and merge cuh it looks better in the timeline and doesnt pollute the tree

File: package.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
 {
-  "name": "slopify",
```
Comment (apucontilde) [inline package.json:2]: dont remove the name bruh

**Review by apucontilde — APPROVED** [review]

**Review by Dishpit — CHANGES_REQUESTED** [review]
this is cool, but it wipes out everything added so far

If you can set up nuxt while also keeping things intact, that'd be best

File: package.json (hunk 1 above)
Comment (niceEli) [reply package.json:2]: that was accidental from create-vite

**Pull request closed without merging**